src/installer.ts

```diff
@@ -114,14 +114,18 @@ async function queryLatestMatch(versionSpec: string) {
 }
 
 async function getAvailableVersions() {
-  const rest = new restm.RestClient("setup-deno");
-  const data =
-    (
-      await rest.get<{ name: string }[]>(
-        "https://denolib.github.io/setup-deno/release.json"
-      )
-    ).result || [];
-  return data.map(v => v.name);
+  // a temporary workaround until a Release API is provided. (#11)

```

Comment:
Nice!

I suddenly found that there is no test case for this function

If you can add the test, then it must be excellent